### PR TITLE
pkg/machine/e2e: wsl stop

### PR DESCRIPTION
**Pull request by baude**
allow the machine stop test to pass.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by baude** [comment]
@n1hility ptal

**Comment by ashley-cui** [comment]
LGTM

File: pkg/machine/e2e/stop_test.go [hunk 1]
```diff
@@ -42,5 +45,6 @@ var _ = Describe("podman machine stop", func() {
 		stopAgain, err := mb.setCmd(stop).run()
 		Expect(err).ToNot(HaveOccurred())
 		Expect(stopAgain).To(Exit((0)))
+		Expect(stopAgain.outputToString()).To(ContainSubstring(fmt.Sprintf("Mach1ine \"%s\" stopped successfully", name)))
```
Comment (ashley-cui) [inline pkg/machine/e2e/stop_test.go:48]:
```suggestion
		Expect(stopAgain.outputToString()).To(ContainSubstring(fmt.Sprintf("Machine \"%s\" stopped successfully", name)))
```

**Pull request merged**